INTRODUCTION.md

```diff
@@ -1950,33 +1950,33 @@ release of librdkafka.
 
 | ApiKey  | Request name        | Kafka max   | librdkafka max          |
 | ------- | ------------------- | ----------- | ----------------------- |
-| 0       | Produce             | 7           | 7                       |
-| 1       | Fetch               | 11          | 11                      |
-| 2       | ListOffsets         | 5           | 1                       |
-| 3       | Metadata            | 8           | 2                       |
-| 8       | OffsetCommit        | 7           | 7                       |
-| 9       | OffsetFetch         | 5           | 1                       |
-| 10      | FindCoordinator     | 2           | 2                       |
-| 11      | JoinGroup           | 5           | 5                       |
-| 12      | Heartbeat           | 3           | 3                       |
-| 13      | LeaveGroup          | 3           | 1                       |
-| 14      | SyncGroup           | 3           | 3                       |
-| 15      | DescribeGroups      | 4           | 0                       |
-| 16      | ListGroups          | 2           | 0                       |
+| 0       | Produce             | 9           | 7                       |
+| 1       | Fetch               | 13          | 11                      |
+| 2       | ListOffsets         | 7           | 2                       |
+| 3       | Metadata            | 12          | 4                       |
+| 8       | OffsetCommit        | 8           | 7                       |
+| 9       | OffsetFetch         | 8           | 1                       |
```

Comment:
Shouldn't this be `7` for librdkafka? https://github.com/confluentinc/librdkafka/blob/master/src/rdkafka_request.c#L971

Reply:
Yes, as this change is included in #3995 I've overlooked this but actually the version 7 was already supported.